src/core/opamStd.ml

```diff
@@ -503,7 +503,8 @@ module OpamSys = struct
 
   let tty_out = Unix.isatty Unix.stdout
 
-  let default_columns = 80
+  let default_columns =
+    try int_of_string (Sys.getenv "COLUMNS") with _ -> 80
```

Comment:
Please use `Env.get` and explicitely catch the exceptions, as in the code below (opam guidelines forbid `try with _`)